source/_integrations/bayesian.markdown

```diff
@@ -149,3 +148,7 @@ binary_sensor:
       prob_given_true: 0.95
 ```
 {% endraw %}
+
+The template is re-evaluated whenever an entity ID that it references changes state. If you use non-deterministic functions like `now()` in the template as it will not be continuously re-evaluated, but only when an entity ID that is referenced is updated. 
```

Comment:
```suggestion
The template is re-evaluated whenever an entity ID that it references changes state. If you use non-deterministic functions like `now()` in the template it will not be continuously re-evaluated, but only when an entity ID that is referenced is updated. 
```